service/src/main/java/uk/gov/hmcts/reform/fpl/service/DocmosisDocumentGeneratorService.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.reform.fpl.config.DocmosisDocumentGenerationConfiguration;
+import uk.gov.hmcts.reform.fpl.enums.DocmosisTemplates;
+import uk.gov.hmcts.reform.fpl.model.common.DocmosisDocument;
+import uk.gov.hmcts.reform.fpl.model.common.DocmosisRequest;
+
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.fpl.config.DocmosisDocumentGenerationConfiguration.DocmosisConfig;
+
+@Service
+public class DocmosisDocumentGeneratorService {
+    private final RestTemplate restTemplate;
+    private final String tornadoUrl;
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+    private final String tornadoAccessKey;
+
+    @Autowired
+    DocmosisDocumentGeneratorService(RestTemplate restTemplate,
+                                     @Value("${docmosis.tornado.url}") String tornadoUrl,
```

Comment:
:-1: I remember you mentioned you don't see much value in the config file, I see what you meant.

The problem is you are now inject twice the Docmosis config params, with that appraoch probably here you should just inject:
DocmosisDocumentGenerationConfiguration

and read the values. You shouldn't use placeholder in two places.